src/App.vue

```diff
@@ -22,3 +24,9 @@ export default {
   },
 };
 </script>
+
+<style scoped>
```

Comment:
@Mitesh2603 we are using tailwind css so for that we don't need style direct apply using class

Reply:
Okay